app/services/current-user.ts

```diff
@@ -92,14 +92,12 @@ export default class CurrentUserService extends Service {
     }
 
     /**
-     * Invalidate the current session and cookie, then redirect to the given URL (or back to the current page).
-     * Returns a promise that never resolves.
+     * Invalidate the current session and cookie
      */
-    async logout(nextUrl?: string) {
```

Comment:
i was concerned we might depend on `nextUrl` -- there's [one place we use it](https://github.com/CenterForOpenScience/ember-osf-web/blob/38d8b6ac89e37b869e67ea9b9fd17c009613abf6/lib/osf-components/addon/components/osf-navbar/auth-dropdown/component.ts#L78), but best i can tell that `redirectUrl` is [only ever `'/goodbye'`](https://github.com/CenterForOpenScience/ember-osf-web/blob/38d8b6ac89e37b869e67ea9b9fd17c009613abf6/lib/osf-components/addon/components/osf-navbar/auth-dropdown/component.ts#L40), and is otherwise referenced [only in a passthrough](https://github.com/CenterForOpenScience/ember-osf-web/blob/38d8b6ac89e37b869e67ea9b9fd17c009613abf6/lib/app-components/addon/components/branded-navbar/template.hbs#L69), so this seems safe

to avoid future confusion (and what i would have assumed would be a typescript error?), maybe remove `redirectUrl` from `OsfNavbar::AuthDropdown` and update it to call just `logout()`?